docs/src/pages/reference/QueryClient.md

```diff
@@ -250,7 +250,7 @@ console.log(state.dataUpdatedAt)
 
 ## `queryClient.setQueriesData`
 
-`setQueriesData` is a synchronous function that can be used to immediately update cached data of multiple queries. Only queries that match the passed queryKey or queryFilter will be updated - no new cache entries will be created. Under the hood, [`setQueryData`](#queryclientsetquerydata) is called for each query.
+`setQueriesData` is a synchronous function that can be used to immediately update cached data of multiple queries by partially matching the query key. Only queries that match the passed queryKey or queryFilter will be updated - no new cache entries will be created. Under the hood, [`setQueryData`](#queryclientsetquerydata) is called for each query.
```

Comment:
`setQueriesData` per se doesn't always partially match the query key. it takes a `filters` argument, and if the filter matches, the data will be set. You can use a filter that doesn't contain a query key. This is the general filters concept of react-query that applies to all functions that have a plural in their name, like `refetchQueries` or `removeQueries`. see: https://react-query.tanstack.com/guides/filters#_top

Reply:
the reason for highlighting partial matching is searchability. This part of documentation does not appear by any word that indicates there is such possibility. All the examples uses `setQueryData` and outdated examples indicates that it would support partial matching. what about `by using filter function or partially matching the query key`?

```suggestion
`setQueriesData` is a synchronous function that can be used to immediately update cached data of multiple queries by using filter function or partially matching the query key. Only queries that match the passed queryKey or queryFilter will be updated - no new cache entries will be created. Under the hood, [`setQueryData`](#queryclientsetquerydata) is called for each query.
```